symfit/contrib/interactive_fit/interactive_fit.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+from symfit import Fit  # Should be ...api import fit. Or something. Relative imports.
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d.axes3d import Axes3D
+import itertools
+
+
+#from pkg_resources import parse_version
+
+#SUPPORTED_VERSION = '0.3.0'
+#SYMFIT_VERSION = symfit.version  # something something.
+#
+#if parse_version(SUPPORTED_VERSION) != parse_version(SYMFIT_VERSION):
+#    raise EnvironmentError("Your symfit version is not supported. YMMV.")
+
+
+class InteractiveFit2D(Fit):
+    """A class that provides a visual_guess method which provides
+    an graphical, interactive way of guessing initial fitting parameters."""
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Can this be split up into multiple methods? This function is way to long, making it hard to read.